kedro_airflow_k8s/operators/node_pod.py

```diff
@@ -174,3 +179,23 @@ def create_security_context(
             if not volume_disabled
             else k8s.V1PodSecurityContext()
         )
+
+    @staticmethod
+    def create_tolerations(
+        tolerations: Optional[List[Dict[str, str]]] = None
+    ) -> List[k8s.V1Toleration]:
+        """
+        Creates k8s tolerations
+        :param tolerations:
+        :return:
+        """
+        if not tolerations:
+            return []
+        return [
+            k8s.V1Toleration(
+                effect=toleration.get("effect"),
+                key=toleration.get("key"),
+                operator=toleration.get("operator"),
+                value=toleration.get("value"))
```

Comment:
Are all tolerations attributes required or some of these are optional? If some are optional, use `toleration.get("operator",None) instead.

Reply:
I think is should be ok:

>>> test = {'xxx': 'test'}
>>> test.get('xxx')
'test'
>>> test.get('yyy')
>>> print(test.get('yyy'))
None
>>> type(test.get('yyy'))
<class 'NoneType'>